HIP/hip-583.md

```diff
@@ -2,221 +2,268 @@
+     * delegatable_contract_id, are not supported.
+     *
+     * At most only one account can ever have a given alias on the network.
+     */
 	bytes alias = 18;
 }
 ```
 
-Additionally, if an account is created with an `ECDSA` public key the alias property should be set automatically at creation to ensure smart contract compatibility from creation. 
+#### Add Public key → alias validation
 
-### Account Creation via alias only (Lazy Create)
-
-![lazy-create-with-address.png](../assets/hip-583/lazy-create-with-address.png)
+If the `alias` is set during `CryptoCreate`, then ownership of the `alias` **must** be proven by the caller
+by signing the transaction with the key represented by the `alias`. It is not necessary for the `key` on the
+newly created account to match the alias, but it **is** necessary for the transaction to be signed by the key
+represented by the `alias`.
 
-The `CryptoCreate` transaction should be expanded to create a new account on the network when only a `public-address` alias is provided with no `public-key`.
+### `CryptoTransfer` to an `evm account alias`
 
-If a `public-key` is provided as an alias the `key` property on the account may be set automatically. When a `public-address` (based on an `ECDSA` key) is provided as an alias the `key` on the account will be left empty and only the `alias` set with a value of the `public-address`.  An account created in this manner will be known as a `Hollow Account` and will only be able to receive HBAR and applicable token transfers, but can not take part in transactions submitted by others requiring its signature.
+TODO: Image needs to be updated
+![lazy-create-with-address.png](../assets/hip-583/lazy-create-with-address.png)
 
-Since Hedera accounts require keys to ensure security, a matching `ECDSA` `public-key` must be provided in a future signed transaction issued by the owner of the `public-address`. Due to the nature of `ECDSA` keys the `public-key` and `public-address` may be extracted from the signature of a transaction signed by the `ECDSA` private key that owns the new account. This process will complete the creation of the account, taking it from a `Hollow Account` to a normal complete account with a valid `public-key`.
-Note, in this case the final value of the account alias will remain as the `public-address` and not the `public-key` as it would be in the `auto-create` case.
+When a user transfers value (hbars or tokens), they may address their destination account using the standard
+account ID format (`shard.realm.num`), or by using an alias format (`shard.realm.alias`). If the alias is an
+`evm address alias`, then the consensus node will first check to see if such an account exists. If it does,
+then it will use the normal rules for transferring the hbars or tokens (including checking `receiverSigRequired`,
+and all other rules). If the account *does not* exist, then the account will be auto-created with the same rules
```

Comment:
`auto-create` is used again here where `lazy-create` was used before.
Are we trying to remove the lazy-create term or is this a mixup?
Clarifying to confirm terminology implications

Reply:
Yes, I was trying to clarify the terms. Auto-creation is automatic-creation, which does not imply full account or hollow account. Any thing that is created automatically falls under the umbrella of auto-creation. Lazy creation means we don't create something until needed -- but actually we *are* creating something, we're creating a hollow account. So I changed the terminology so that auto-creation encompasses anything that is automatically created, and hollow creation is specifically the creation of a hollow account.

Reply:
Makes sense.
All DI frameworks would agree also :)